pkgs/development/arduino/arduino-mk/default.nix

```diff
@@ -11,8 +11,16 @@ stdenv.mkDerivation rec {
     sha256 = "0flpl97d2231gp51n3y4qvf3y1l8xzafi1sgpwc305vwc2h4dl2x";
   };
 
-  phases = ["installPhase"];
-  installPhase = "ln -s $src $out";
+  nativeBuildInputs = with python2Packages; [ wrapPython ];
```

Comment:
Can you please use python 3 if possible? We are trying to deprecate python 2.

Reply:
Done